test/connection.browser.spec.ts

```diff
@@ -41,7 +41,7 @@ describe('connection browser tests', () => {
       break;
     }
     expect(responsed).to.be.true();
-  });
+  }).timeout(54321);
```

Comment:
what's the point of such a big time out?

Reply:
@John-LittleBearLabs I thought this was removed in your PR?